### DISCUSSION TITLE
My github page is not loading its css nor its js. What do i do?

### DISCUSSION THREAD

**Discussion by greeenboi** [community/community#46543] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

<p>I made a website using Three.js, and when I run it locally, it works fine but when I added my files to the [repo]https://github.com/greeenboi/greeenboi.github.io) it shows only the html parts and nothing else. 
<br>
How do I fix this issue? and if Github is not meant to run three.js where else can I host it for cheap if not free?</p>

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @greeenboi 

GitHub Pages does not provide support for scripts of type "module". It is designed to provide a static hosting platform for websites and does not offer support for dynamic features such as `module` type scripts.

[Vercel ](https://vercel.com/dashboard) and [Netlify ](https://www.netlify.com/) are alternative hosting platforms that offer support for "module" type scripts

**Comment by greeenboi** [comment]
Thank you very much @SilasRodrigues19 .
